libfreerdp/cache/bitmap.c

```diff
@@ -278,6 +282,108 @@ void bitmap_cache_register_callbacks(rdpUpdate* update)
 	}
 }
 
+int bitmap_cache_load_persistent(rdpBitmapCache* bitmapCache)
+{
```

Comment:
AFAICT this function is never called and does not much (its only function would be to check that the cache file "loadable"), shall we remove it ?